cocos/editor-support/cocostudio/ActionTimeline/CCActionTimeline.cpp

```diff
@@ -355,6 +373,52 @@ void ActionTimeline::emitFrameEvent(Frame* frame)
     }
 }
 
+void ActionTimeline::addFrameEndCallFunc(int frameIndex, const std::string& funcKey, std::function<void()> func)
+{
+    if (func != nullptr)
+    {
+        _frameEndCallFuncs[frameIndex][funcKey] = func;
+    }
+}
+
+void ActionTimeline::removeFrameEndCall(int frameIndex, const std::string& funcKey)
```

Comment:
Please keep API name consistency: rename `ActionTimeline::removeFrameEndCall` to `ActionTimeline::removeFrameEndCallFunc`